src/components/molecules/Personalisation/Personalisation.tsx

```diff
@@ -0,0 +1,45 @@
+import concatClassNames from '@/src/utils/concatClassNames';
+import styles from './Personalisation.module.scss';
+import content from '@/src/content/content';
+import { Details } from 'nhsuk-react-components';
+
+const personalisationContent = content.components.personalisationComponent;
+
+export type PersonalisationType = {
+  isOpen?: boolean;
+};
+
+export function Personalisation({ isOpen }: PersonalisationType) {
+  return (
+    <>
+      <h2
+        className='nhsuk-heading-m nhsuk-u-margin-top-4'
+        data-testid='personalisation-header'
+      >
+        {personalisationContent.header}
+      </h2>
+      <Details data-testid='personalisation-details'>
+        <Details.Summary data-testid='personalisation-summary'>
+          {personalisationContent.details.title}
+        </Details.Summary>
+        <Details.Text data-testid='personalisation-text'>
+          <p>{personalisationContent.details.text1}</p>
+          <code className={styles.codeBackground}>
+            {personalisationContent.details.codeBlockText}
+          </code>
+          <p className='nhsuk-u-margin-top-4'>
+            {personalisationContent.details.text2}
+            <span className={styles.inlineText}>
```

Comment:
Using a `span` makes it so there is no space between `text2` and `text3`. Which looks like

```
NHS Notify gets data from PDS to populate personalisation fields.You can use:
```

Perhaps remove text3 and put the words into text2? 

I think this could be spotted in a snapshot test.

Reply:
There is space between them and that is why I have put a style on the span to make it `inline-block` which is better than having a line break that comes from the design

Reply:
You could stick each text block in a `<div>`.

Reply:
@mark-r-bjss that won't be a good practice also and it won't conform with the design. Putting them in a `div` would also mean additional styles which is not needed.

Can we have a call to see this.

Reply:
Sorry, I should've explained what I meant better. There is no space between the fullstop at the end of `text2` and the beginning of `text3`.

The inline-block is forcing the span onto a new line due to the width of the container, which makes it looks like there is a space between the fullstop. If the width changes and the span will wrap back up like so;

![image](https://github.com/NHSDigital/nhs-notify-web-template-management/assets/131893816/f278dfff-b966-4aa0-baeb-20fa8ee02b0e)

The inline-block probably isn't needed.

If you move the personalisation element under the main div content or go into mobile view it's a bit easier to see what's happening. 

![image](https://github.com/NHSDigital/nhs-notify-web-template-management/assets/131893816/17315796-6064-42a8-a001-841cf48af462)

Reply:
Thanks for the call @bhansell1 will raise it with the R&D team

Reply:
Fixed now